### ISSUE TITLE
Timecode Remaining hour/minute variables are incorrect

### ISSUE THREAD

**Issue by swampcat** [bitfocus/companion-module-bytehive-playoutbee#4]
The variables ```remaining_hh``` and ```remaining_mm``` are being calculated incorrectly. For example, if the remaining timecode returned is 2373.91669 (39:33), the variable ```remaining_hh``` contains 06, not 00, and ```remaining_mm``` is 03, not 39. The problem appears to be at lines 43-45 in variables.js where 360 is used as the divisor to calculate the hours, not 3600. The code should be

```
const hh = parseInt(remainder / 3600)

remainder = remainder % 3600
```

**Comment by estilles** [comment]
@swampcat 
Thanks for reporting this. Fixing ASAP.

**Comment by estilles** [comment]
Fixed via 277323c2208235c55b8bec1a1d5d1dfd5ecb088f

**Comment by estilles** [comment]
Fix available in the latest Companion beta build.
